### PR TITLE
Component completed task

### PR DESCRIPTION
**Pull request by Divya-Manivannan1**
Component created with MUI. Tests and stories created with RTL and storybook

### CODE REVIEW

File: src/components/CompletedTask/CompletedTask.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  styled,
+} from "@mui/material";
+
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+
+interface ExpandMoreProps extends IconButtonProps {
+  expand: boolean;
+}
+
+const ExpandMore = styled((props: ExpandMoreProps) => {
```
Comment (Charlie-robin) [inline src/components/CompletedTask/CompletedTask.tsx:22]: Can you style this without the use of styled components?

File: src/components/CompletedTask/CompletedTask.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+  return (
+    <Card className="completed-task">
+      <CardHeader
+        className="completed-task__header"
+        action={
+          <ExpandMore
+            expand={expanded}
+            onClick={handleExpandClick}
+            aria-expanded={expanded}
+            aria-label="show more"
```
Comment (Charlie-robin) [inline src/components/CompletedTask/CompletedTask.tsx:63]: You should be able to add your styles from the styled component here.

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
I think you need to pull from main and merge it into your branch so it is up to date.

 I have added a couple of points to have a look at, @ample-samples will do the same. Reach out if you need any clarity on them.

File: src/components/CompletedTask/CompletedTask.test.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+      image={
+        "https://img.freepik.com/premium-photo/running-man-silhouette-sunset-time-sport-active-life-concept_221513-1606.jpg"
+      }
+    />
+  );
+  const arrowIcon = screen.getByTestId("ExpandMoreIcon");
+  fireEvent.click(arrowIcon);
+  expect(await screen.findByText("Media")).toBeInTheDocument();
+  expect(await screen.findByText("Run as fast as you can")).toBeInTheDocument();
+  expect(await screen.findByAltText("Run 5k")).toBeInTheDocument();
```
Comment (ample-samples) [inline src/components/CompletedTask/CompletedTask.test.tsx:61]: Can you restructure these expect statements to use `queryByText` and store these queries in a variable as you've done in the other tests

**Review by ample-samples — CHANGES_REQUESTED** [review]
Just a small change to go with Charlie's review

File: src/components/CompletedTask/CompletedTask.test.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+
+import CompletedTask from "./CompletedTask";
+
+it("should render the Completed Task component", () => {
```
[the remark this answers is not in the document]
Comment (Divya-Manivannan1) [reply src/components/CompletedTask/CompletedTask.test.tsx:5]: Added describe

File: src/components/CompletedTask/CompletedTask.test.tsx (hunk 3 above)
Comment (Divya-Manivannan1) [reply src/components/CompletedTask/CompletedTask.test.tsx:61]: We have taken them out as seperate variables, but we are still using FindbyText. The clicking action has some animation associated with it, so we need to use await to wait for these elements to load before testing them.

File: src/components/CompletedTask/CompletedTask.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import CompletedTask from "./CompletedTask";
+
+/** The CompletedTask component has 5 props: taskHeading(string), category(string), points(number), description(string) and image(url as a string). The image prop is optional.*/
```
[the remark this answers is not in the document]
Comment (Divya-Manivannan1) [reply src/components/CompletedTask/CompletedTask.stories.tsx:4]: done :)

File: src/components/CompletedTask/CompletedTask.tsx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+        }
+      />
+
+      <Collapse in={expanded} timeout="auto" unmountOnExit>
+        <CardContent>
+          <Typography paragraph className="completed-task__description">
+            {description}
+          </Typography>
+
+          {image ? (
```
[the remark this answers is not in the document]
Comment (Divya-Manivannan1) [reply src/components/CompletedTask/CompletedTask.tsx:96]: done :)

File: src/components/CompletedTask/CompletedTask.tsx (hunk 1 above)
Comment (Divya-Manivannan1) [reply src/components/CompletedTask/CompletedTask.tsx:22]: We've removed the styled element

**Review by Charlie-robin — APPROVED** [review]

**Pull request merged**